packages/yoastseo/src/scoring/assessors/assessor.js

```diff
@@ -86,6 +86,9 @@ class Assessor {
 	 * @returns {boolean} Whether or not the Assessment is applicable.
 	 */
 	isApplicable( assessment, paper, researcher ) {
+		if ( typeof assessment.isApplicable === "undefined" ) {
+			return true;
+		}
```

Comment:
I added this condition so that I could remove the `isApplicable` method from the previously used keyphrase assessment, instead of having a method that always returns `true`. The previously used keyphrase assessment is not based on the `Assessment` class that has the `isApplicable` method that returns `true` by default.